requirements.txt

```diff
@@ -1,5 +1,5 @@
 traitlets>=4.2
-jupyter_client>=6.0.0
```

Comment:
Let's wait for 6.1.0 before we merge so we have proper versions in the dependency files

Reply:
ok should be changeable to 6.1.0 now